civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -326,9 +328,9 @@
     }
 
     var calendarIcon = '<span class="input-group-addon pointer"><i class="fa fa-calendar"></i></span>';
-    $('#edit-access-min').datepicker();
-    $('#edit-access-min').after(calendarIcon);
-    $('#edit-access-max').datepicker();
-    $('#edit-access-max').after(calendarIcon);
+    $('#edit-access-from').datepicker();
```

Comment:
why are the ids different? Were they wrong to begin with? Were they changed in this PR? The description says nothing about it

Reply:
Explained in the description.